examples/lax_root_batch.py

```diff
@@ -8,38 +16,88 @@
 
 import root_pandas
 
+
 parser = argparse.ArgumentParser(description="Create lichen ROOT files with lax")
 
 parser.add_argument('--run_number', dest='RUN_NUMBER',
                     action='store', required=True, type=int,
-                    help='Run number to process')
+                    help='Run number to process (-1 for MC)')
 
 parser.add_argument('--pax_version', dest='PAX_VERSION',
                     action='store', required=True,
                     help='pax version to process')
 
+parser.add_argument('--minitree_path', dest='MINITREE_PATH',
+                    action='store', required=True,
+                    help='Path to hax minitrees')
+
+parser.add_argument('--filename', dest='FILENAME',
+                    action='store', required=False,
+                    help='Name of pax file (without .root)')
+
+parser.add_argument('--output_path', dest='OUTPUT_PATH',
+                    action='store', required=False, default='',
+                    help='Name of output file (without .root)')
+
 args = parser.parse_args(sys.argv[1:])
 
-hax.init(experiment='XENON1T',
-         pax_version_policy=args.PAX_VERSION,
-         minitree_paths = ['.','/project2/lgrandi/xenon1t/minitrees/pax_v6.4.2'],
-         #make_minitrees = False
-)
+PAX_VERSION_POLICY = args.PAX_VERSION
+RUN_NUMBER = args.RUN_NUMBER
+MINITREE_NAMES = ['Fundamentals', 'Basics', 'TotalProperties',
+                  'Extended', 'TailCut', 'Proximity']
+OUTPUT_PATH = args.OUTPUT_PATH
+
+LAX_LICHENS = [sciencerun0.AllEnergy(),
+               sciencerun0.LowEnergyRn220(),
+               sciencerun0.LowEnergyAmBe(),
+               sciencerun0.LowEnergyBackground()]
+
+TREENAME = 'tree'
+
+# MC
+if args.RUN_NUMBER < 0:
+
+    # No run dependent sims yet
+    PAX_VERSION_POLICY = 'loose'
+
+    # Use filename instead of run number
+    RUN_NUMBER = args.MINITREE_PATH+'/'+args.FILENAME
```

Comment:
This is changed in the [new bin script](https://github.com/XENON1T/lax/blob/d95be1f6fd2f230c2e26fd95ae0724e5c8eaf618/bin/laxer) (this example script will be removed).